source/opt/loop_unroller.cpp

```diff
@@ -736,8 +746,9 @@ void LoopUnrollerUtilsImpl::CloseUnrolledLoop(Loop* loop) {
 
   // Remove the final backedge to the header and make it point instead to the
   // merge block.
-  state_.previous_latch_block_->tail()->SetInOperand(
+  state_.previous_latch_block_->terminator()->SetInOperand(
```

Comment:
Is this a NOP change?

Reply:
In the next line I needed the same instruction, but I needed an `Instruction*` instead of a `BasicBlock::Itertator`.  Just keep the two consistent.  I probably should have created a new variable that is used in both places.